cloud/endagaweb/templates/dashboard/tower_detail/deregister.html

```diff
@@ -71,7 +92,8 @@ <h4 class='modal-title'>
   {% include "dashboard/tower_detail/nav.html" with uuid=tower.uuid active_tab='deregister'%}
 
   {% if endaga_version > '00000.00003.00019' or status == 'no-data' %}
```

Comment:
I don't think this `status == 'no-data'` check actually does what it's supposed to, which is to allow deregistration if the tower has never provided a software version during checkin: the UI currently behaves in the exact opposite way, i.e., no data = deregistration not permitted. It would be good to fix this as part of this PR, if you can figure out why `status` is not as expected.